bpf/tests/Makefile

```diff
@@ -6,11 +6,10 @@ include ../../Makefile.defs
 MAKEFLAGS += -r
 
 CLANG ?= clang
-LLC   ?= llc
 
 FLAGS := -I$(ROOT_DIR)/bpf/include -I$(ROOT_DIR)/bpf -D__NR_CPUS__=$(shell nproc --all) -O2 -g
 
-CLANG_FLAGS := ${FLAGS} --target=bpf -std=gnu99 -nostdinc -emit-llvm
+CLANG_FLAGS := ${FLAGS} --target=bpf -std=gnu99 -nostdinc -march=bpf
```

Comment:
Is `-march=bpf` required in this case and covering sth which `--target=bpf` itself doesn't address?

Reply:
Good point, we also don't pass this when compiling from Go. Removed.